test/gatekeeper/test-mutations.yaml

```
@@ -0,0 +1,35 @@
+---
+apiVersion: mutations.gatekeeper.sh/v1beta1
+kind: Assign
+metadata:
+  name: allow-privilege-escalation
+spec:
+    match:
+      scope: Namespaced 
+      kinds:
+        - apiGroups: ["*"]
+          kinds: ["Pod"]
+    applyTo:
+    - groups: [""]
+      kinds: ["Pod"]
+      versions: ["v1"]
+    location: "spec.containers[name:*].securityContext.allowPrivilegeEscalation"
+    parameters:
+      pathTests:  
```

```suggestion
      pathTests:
```
Remove whitespace at EOL